articles/language/file-structure.md

```diff
@@ -50,6 +50,7 @@ References to types or callables declared further down in a file are resolved pr
 there is no need for the type, operation, or function declaration to precede a reference to it.
 
 ## Open Directives
+[link text](#open-directives)
```

Comment:
It looks like this link is showing up in the rendered output?
![image](https://user-images.githubusercontent.com/31516/57169870-390edd00-6dbe-11e9-9b65-31ff574be8a5.png)